src/components/repository-profile.component.js

```diff
@@ -141,6 +143,10 @@ export const RepositoryProfile = ({
         {emojifyText(repository.description) || ' '}
       </Text>
 
+      <Text style={styles.languageInfoTitle}>
+        {subscribed ? '(watching)' : undefined}
```

Comment:
I agree regarding the languages, I will update that.

Since the style did not need to change much, I decided to use a style that was already created. In any case, I can copy the contents into its own and call it `watchingLabel`.

How would `null` be better than `undefined` in this situation? Both end up rendering nothing to the DOM (And I think the same goes for `false`)